selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -111,6 +112,11 @@ public Boolean getEmulator() {
     return getBooleanCapability(EMULATOR);
   }
 
+  public boolean getUseJUnitRunner() {
```

Comment:
Let's call this something like `getUseJUnitBootstrap`? I'm afraid folks will be confused by the `Runner` part as selendroid is related to testing. We just happen to use a weird hack to bootstrap the selendroid server.